stacker/blueprints/base.py

```diff
@@ -402,6 +406,44 @@ def render_template(self):
         version = hashlib.md5(rendered).hexdigest()[:8]
         return (version, rendered)
 
+    def parse_user_data(self, raw_user_data):
+        """Translate a userdata file to into the file contents.
+
+        It supports referencing template variables to create userdata
+        that's supplemented with information from the data, as commonly
+        required when creating EC2 userdata files. Automatically, encodes
+        the data file to base64 after it is processed.
+
+         Args:
+            raw_userdata (str of userdata):
```

Comment:
Just `str` instead of `str of userdata`.  Also, make sure you put the return statement in the docstring as well - see: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html